debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleDdlParser.java

```diff
@@ -213,23 +213,34 @@ else if (ctx.datatype().native_datatype_element().FLOAT() != null ||
                         .jdbcType(Types.FLOAT)
                         .type("FLOAT")
                         .length(126);
+
+                    // TODO float's precision is about bits not decimal digits; should be ok for now to over-size
+                    if (precisionPart != null) {
+                        setPrecision(precisionPart, columnEditor);
+                    }
                 }
                 else if (ctx.datatype().native_datatype_element().REAL() != null) {
                     columnEditor
                         .jdbcType(Types.FLOAT)
                         .type("FLOAT")
                         .length(63);
+
```

Comment:
Does `REAL` really have a precision part?
> The REAL datatype is a floating-point number with a binary precision of 63, or 18 decimal.

Reply:
`REAL` has a fixed precision, i.e. you can't customize it.

Reply:
But does not the following code try to work with precision?

Reply:
I'm not quite following. Can you ellaborate a bit? We set the length above (ignoring the fact for now that the value is too large, as 63 is the number of bits, not the number decimal digits, but this shouldn't matter really for now).

Reply:
See line https://github.com/gunnarmorling/debezium-incubator/blob/0d5f5cfa7eca9899a1645db294e27b01f551fecc/debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleDdlParser.java#L235 There is no precision part according to the documentation yet you expect that it might arrive

Reply:
Aaaah, gotcha. So those lines L235...238 are superfluous. Thanks! I'll remove them.

Reply:
Btw. another step will be to retrofit this all into the basic parser structure foreseen by Roman. I had hoped to be able to do it for the Alpha1, but it seems it'll not be done by then.